bot_instructions.rb

```diff
@@ -0,0 +1,11 @@
+class BotInstructions
+  @@instructions = YAML.load_file('instructions.yml')
```

Comment:
Yeah, you can do that. It's really a only one actually useful usecase of class variables in Ruby :) Other on that, it considered harmful and trying to be avoided as much as possible.

Next time you can do something like this: 
```ruby
module BotInstruction
  extend self
  
  def instructions
    @instructions ||= YAML.load_file('./instructions.yml')
  end

  def replies
    instructions['replies']
  end

  def commands
    instructions['commands']
  end
end
```